code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -846,9 +846,10 @@ var/list/not_resolved_in_attackby_objects = list(/obj/structure/chemical_dispens
 	icon_state = "fueltank_large"
 	volume = 250
 	density = TRUE
+	flags = OPENCONTAINER
```

Comment:
and if any flags was set before you will clear them there!

Reply:
This was intentional, when you closed the gas tanks you made them impossible to refill the ones attached to vehicles.

Reply:
If you need them open, then put to New() subproc 
flags |= OPENCONTAINER
but not do it with "=" operator, because you work with flags, not with single variable

Reply:
My bad, I'm unfamiliar with this new format. If it's done like that it will still effect the gas tanks that descend from it?

Reply:
Of course. But with the exception of those objects that had flags directly marked during mapping. But usually, mappers don't go into variables and leave default values, which are defined both in the object definition itself and then set in the New() procedure.
In general, operations with flag variables are carried out because operations cannot be included in definitions. For example, it is not allowed to introduce a construction like "weight=mass*G" into the definition of an object. Therefore, if we want exactly this value, then we enter "weight=0" in the object definition, and already in the New () procedure we enter the type construction:
    ..() // this is for the object's parent procedure to be processed first
    weight=mass*G // value is directly assigned here

sometimes if the parent object uses sleep(t) in its New(), then we also have to implement sleep(t+1) in the child object.

Reply:
P.S. If mappers set something in variables, than New() proc will inherit them too.